src/libLookup/Lookup.cpp

```diff
@@ -772,6 +772,13 @@ void Lookup::RetrieveDSBlocks(vector<DSBlock>& dsBlocks, uint64_t& lowBlockNum,
 
   uint64_t curBlockNum =
       m_mediator.m_dsBlockChain.GetLastBlock().GetHeader().GetBlockNum();
+
+  if (INIT_BLOCK_NUMBER == curBlockNum) {
+    LOG_GENERAL(INFO,
```

Comment:
Suggest to use WARNING for this kind of error that would lead to abnormal behavior.